docker/tools/Dockerfile

```diff
@@ -5,11 +5,12 @@ LABEL name="kanister-tools" \
     version="${kan_tools_version}" \
     release="${kan_tools_version}" \
     summary="Microservice for application-specific data management for Kubernetes" \
-    maintainer="Tom Manville<tom@kasten.io>"
+    maintainer="Tom Manville<tom@kasten.io>" \
     description="Kanister tools for application-specific data management"
 
 COPY --from=restic/restic:0.9.5 /usr/bin/restic /usr/local/bin/restic
-COPY --from=kanisterio/kopia:alpine-5261c72 /kopia/kopia /usr/local/bin/kopia
+# kopia alpine-e77c6c1 image
+COPY --from=kanisterio/kopia@c471682028368cca4bb6c3f27454ebaa3b53f7281b925a3abad74a2e65e7d006 /kopia/kopia /usr/local/bin/kopia
```

Comment:
Using SHA256 digest instead of image tag.

Reply:
wfm